src/qutip_tensorflow/core/data/convert.py

```diff
@@ -0,0 +1,33 @@
+import qutip as qt
```

Comment:
I would argue there's no reason to abbreviate `qutip`.  It's short already, and it just hurts readability.